Server/ChatService/routes/chat.py

```
@@ -329,7 +343,7 @@ async def send_fcm_async():
                         print(f"[chat.ws] FCM notification failed: {e}")
                 
                 # Fire and forget - only if WebSocket delivery failed
-                if not delivered:
+                if not recipient_delivered:
                     asyncio.create_task(send_fcm_async())
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)